{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/templates/email/password_reset_mail.tpl

```diff
@@ -0,0 +1,39 @@
+{% raw %}
+{% extends "mail_templated/base.tpl" %}
+{% load i18n %}
+
+{% block subject %}
+Reset your {{ site.name }} Password!
+{% endblock %}
+
+{% block body %}
+
+{% blocktrans %}You're receiving this email because you requested a password reset
+for your user account.{% endblocktrans %}
+
+{% trans "Please go to the following page and choose a new password:" %}
+{% block reset_link %}
+{{ frontend_domain }}/{{ url }}
```

Comment:
Let's hv the full url coming from view itself. Let's not build the url in the template.